### PR TITLE
[ja] Update README-ja.md

### PR DESCRIPTION
**Pull request by nasa9084**
Update README-ja.md following README.md

Fix: #33122

### CODE REVIEW

File: README-ja.md [hunk 1]
```diff
@@ -4,6 +4,9 @@
 
 このリポジトリには、[KubernetesのWebサイトとドキュメント](https://kubernetes.io/)をビルドするために必要な全アセットが格納されています。貢献に興味を持っていただきありがとうございます！
 
+- [ドキュメントにコントリビュートする](#contributing-to-the-docs)
+- [READMEの翻訳](#localization-readmemds)
```
Comment (t-inu) [inline README-ja.md:8]:
```suggestion
- [ドキュメントに貢献する](#contributing-to-the-docs)
- [翻訳された`README.md`一覧](#localization-readmemds)
```
These are links to the headings in this page.
I think the link text would be better to match the heading text.

And, it is necessary to set the IDs at the headings to be accessible.
```
## ドキュメントに貢献する {#contributing-to-the-docs}
```
```
## 翻訳された`README.md`一覧 {#localization-readmemds}
```

Comment (nasa9084) [reply README-ja.md:8]: ah thank you, let me add ids

**Pull request merged**